### PR TITLE
chore: Added `star on GitHub` image 🌟🚀

### PR DESCRIPTION
**Pull request by Nishitbaria**
## Related Issue

Issue : #1810

## Description

This PR proposes an improved logo type for the ProjectHunt Project to enhance user engagement and encourage GitHub contributions. The updated logo prominently showcases "Start Contribution" ✨ at the top, followed by "Star on GitHub" 🤝 to prompt users to interact with the repository. Additionally, a dynamic element indicates the total number of stars the project has received, giving users a quick overview of its popularity 🌟.

Key Changes:

- **Responsive Design:** The logo is now responsive, ensuring a seamless experience across various screen sizes.

**Code Changes:**

```html
<div className="github-button p-2 bg-white rounded-lg max-w-[170px] mx-auto">
  <a
    className="github"
    href="https://github.com/priyankarpal/ProjectsHut"
    target="_blank"
    rel="noreferrer"
  >
    <img
      src="https://github.com/priyankarpal/ProjectsHut/assets/88102392/0fa9e8f0-349d-4cc8-8e01-1b6e2c27dbbb"
      alt="Github-Share-Button"
    />
  </a>
</div>
```

## Screenshots

image

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
add the image to the right position 
image

**Comment by Nishitbaria** [comment]
https://github.com/priyankarpal/ProjectsHut/assets/85815172/82e2caa9-171a-4432-8f36-ccb3ec767ce9

@priyankarpal Please check the changes

**Review by priyankarpal — CHANGES_REQUESTED** [review]
please change. align the license & docs heading 

Group 2

**Comment by Nishitbaria** [comment]
image
I have fix alignment

**Comment by Nishitbaria** [comment]
@priyankarpal Check

File: components/Footer.tsx [hunk 1]
```diff
@@ -118,17 +140,42 @@ function Footer() {
...
             </ul>
+            <div className="mt-[25px]">
+              <div className="github-button p-2 bg-white rounded-lg max-w-[170px] mx-auto  ">
+                <a
+                  className="github"
+                  href="https://github.com/priyankarpal/ProjectsHut"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <img
```
Comment (priyankarpal) [inline components/Footer.tsx:172]: use `<Image>`

https://nextjs.org/docs/pages/api-reference/components/link

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: components/Footer.tsx [hunk 2]
```diff
@@ -118,17 +140,42 @@ function Footer() {
...
+                          {link.name}
+                        </Link>
                       </li>
                     ))}
                   </ul>
                 </div>
               ))}
             </ul>
+            <div className="mt-[25px]">
+              <div className="github-button p-2 bg-white rounded-lg max-w-[170px] mx-auto  ">
```
[the remark this answers is not in the document]
Comment (Nishitbaria) [reply components/Footer.tsx:165]: Thanks for Replying!!! That was just a className I though I would used but not needed

I will resolve this

File: components/Footer.tsx [hunk 3]
```diff
@@ -118,17 +140,42 @@ function Footer() {
...
+                <a
```
[the remark this answers is not in the document]
Comment (Nishitbaria) [reply components/Footer.tsx]:
```javascript
<Link
                  href="https://github.com/priyankarpal/ProjectsHut"
                  target="_blank"
                  rel="noreferrer"
                  aria-label="GitHub repository"
                >
                  <Image
                    src="https://github.com/priyankarpal/ProjectsHut/assets/88102392/0fa9e8f0-349d-4cc8-8e01-1b6e2c27dbbb"
                    alt="Github-Share-Button"
                    width={144}
                    height={46}
                  />
                </Link>
```


Fixed

File: components/Footer.tsx (hunk 1 above)
Comment (Nishitbaria) [reply components/Footer.tsx]:
```javascript
                  <Image
                    src="https://github.com/priyankarpal/ProjectsHut/assets/88102392/0fa9e8f0-349d-4cc8-8e01-1b6e2c27dbbb"
                    alt="Github-Share-Button"
                    width={144}
                    height={46}
                  />
                
```

This also Fixed

**Comment by Nishitbaria** [comment]
@priyankarpal Fixed you reviews

**Comment by priyankarpal** [comment]
@Nishitbaria in the future, kindly refrain from using these types of commit messages, as they do not adhere to the valid format. If a user reviews the repository, they might encounter difficulties in understanding the changes made by the contributor.

image

image

**Review by priyankarpal — APPROVED** [review]

**Comment by Nishitbaria** [comment]
@priyankarpal Man I deserve level 2 atleast

**Comment by priyankarpal** [comment]
> @priyankarpal Man I deserve level 2 atleast lol 😂

I believe that it was a simple change 🙂

**Comment by Nishitbaria** [comment]
@priyankarpal Ohh I see Thanks for Merging 🙃

**Pull request merged**